packages/components/src/components/hds/app-frame/index.hbs

```diff
@@ -1,25 +1,24 @@
-{{! @glint-nocheck: not typesafe yet }}
 {{!
   Copyright (c) HashiCorp, Inc.
   SPDX-License-Identifier: MPL-2.0
 }}
 <div class="hds-app-frame" ...attributes>
   {{#if this.hasHeader}}
-    {{yield (hash Header=(component "hds/app-frame/parts/header"))}}
+    {{yield (hash Header=(component "hds/app-frame/header"))}}
```

Comment:
These `parts` should be preserved, as they reflect the structure of this component
```suggestion
    {{yield (hash Header=(component "hds/app-frame/parts/header"))}}
```